torch/csrc/jit/codegen/cuda/executor.cpp

```diff
@@ -52,8 +53,11 @@ void FusionExecutor::debugCompileFusionFromStr(
   FusionGuard fg(&fusion_);
   options_ = options;
 
-  const char* debug_env = std::getenv("PYTORCH_CUDA_FUSER_DEBUG");
-  if (debug_env && atoi(debug_env)) {
+  if (isDebugDumpEnabled(DebugDumpOption::FusionIr)) {
+    fusion->print();
```

Comment:
Can we do `printMath` instead of print? An option for it as well would be okay, but I rarely debug with fusion->print, it's too verbose to dig through.

Reply:
I'll switch to printMath(), but wouldn't that miss potentially relevant information out?